roles/skydive/tasks/provision.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Login As Super User
```

Comment:
this task is no longer needed since login is always done at the playbook level

Reply:
Good to know! I will open a PR to remove this from the multus deployment also.

Reply:
Done